website/blog/2023-05-30-0.72-metro-package-exports-symlinks.md

```diff
@@ -0,0 +1,191 @@
+
+:::info
+Please update your config file to following [format](https://github.com/facebook/react-native/blob/76a42c292de838a0dd537935db792eaa81410b9b/packages/react-native/template/metro.config.js). You can also follow the [upgrade-helper](https://react-native-community.github.io/upgrade-helper/?from=0.71.8&to=0.72.0).
+
+These format changes to `metro.config.js` will become required in 0.73. For 0.72, we will log a warning if not updated.
+:::
+
+This moves control over extending the base React Native Metro config into your project, and we’ve cleaned up the leftover defaults. In addition, this means that standalone Metro CLI commands, such as `metro get-dependencies`, will now work.
+
+#### Symlink support (Beta)
+
+Symlink support continues to be one of the top-requested features in Metro and, in React Native 0.72, we are happy to announce pre-stable support for it.
+
+Symlink support enables React Native to work transparently with monorepo setups and pnpm, removing the need for workarounds. See [Enabling pre-stable features](/blog/2023/05/30/0.72-metro-package-exports-symlinks#enabling-pre-stable-features) to use in your app.
+
+#### Package Exports support (Beta)
+
+[Package Exports](https://nodejs.org/api/packages.html#exports) is the modern alternative to the package.json "main" field and provides new capabilities for npm packages to define their public API and target React Native.
+
+By enabling Package Exports support in your Metro config, your app will be compatible with the wider JavaScript ecosystem, including via the new "react-native" community condition. See [Enabling pre-stable features](/blog/2023/05/30/0.72-metro-package-exports-symlinks#enabling-pre-stable-features) to use in your app.
+
+:::tip
+See the [dedicated blog post](https://reactnative.dev/blog/2023/05/08/package-exports-support) to learn more about this feature and our plans for stable rollout.
+:::
+
+#### Enabling pre-stable features
+
+To enable these new Metro features in your project, update your app’s `metro.config.js` file and set either the `resolver.unstable_enableSymlinks` or `resolver.unstable_enablePackageExports` options.
+
+```js
+const config = {
+  // ...
+  resolver: {
+    unstable_enableSymlinks: true,
+    unstable_enablePackageExports: true,
+  },
+};
+```
+
+### Developer Experience Improvements
+
+#### No more redboxes with invalid style properties
+
+Prior to this release, providing an invalid style property in StyleSheet would result in a redbox. This is a high signal error that disrupts the developer workflow for a relatively low-risk error.
+
+In 0.72, we’ve relaxed this expectation to fail silently, like providing an invalid CSS property in the browser.
```

Comment:
We also were able to turn a couple of these into type-checking errors so your editor/typechecker flags them for you (vs the runtime error).